databases.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"time"
+)
+
+// Инициализация базы данных
+func initDb(config config) error {
+	var migration bool
+
+	// Проверяем, существует ли файл базы данных
+	if _, err := os.Stat(config.DbFilePath); os.IsNotExist(err) {
+		migration = true
+	}
+
+	// Открываем базу данных
+	var err error
+	db, err = sql.Open("sqlite", config.DbFilePath)
+	if err != nil {
+		return fmt.Errorf("ошибка открытия базы данных: %v", err)
```

Comment:
При выводе в лог `%v`, а при оборачивании `%w` (wrap)
```suggestion
		return fmt.Errorf("ошибка открытия базы данных: %w", err)
```

Reply:
Хорошо бы везде поправить